src/geography/country.ts

```diff
@@ -9,4 +9,253 @@ export default interface Country {
 export interface Region {
     code: string;
     name: string;
+    id?: number;
 }
+
+export interface GetCountryResponse {
+    data: Country[];
+}
+
+export interface UnitedStatesCodes {
+    name: string;
+    abbreviation: string;
+}
+
+export const UNITED_STATES_CODES = [
```

Comment:
It might be better to create separate file for this constant

Reply:
moved to separate file